packages/orchestrator/template-manager.proto

```diff
@@ -47,10 +48,18 @@ message TemplateCreateRequest {
   TemplateConfig template = 1;
 }
 
+enum LogLevel {
+    Debug = 0;
+    Info = 1;
+    Warn = 2;
+    Error = 3;
+}
+
 message TemplateStatusRequest {
   string templateID = 1;
   string buildID = 2;
   optional int32 offset = 3;
+  optional LogLevel level = 4;
```

Comment:
Maybe name it something like `minLevel` to make obvious it's used for filtration here?